### ISSUE TITLE
Sub-lists should translate into something

### ISSUE THREAD

**Issue by cognivore** [n0x1m/md2gmi#2]
Just like with other entities in Markdown that don't have a counterpart in Gemtext, we need to come up with some translation ourselves.

What I normally do by hand is when I find a non-flat list is I translate each depth level into a level of outline. It is not a generic solution and actually I don't think it should be implemented like this.

I think an acceptable translation would be to insert line breaks and pad each depth level with a number, this way

```
1. Introductory questions about overall background.
	1. Music, computers and work at REDACTED.
	2. Working with humans in non-tech
		1. at work with REDACTED.
		2. in an activist / advocacy group with REDACTED.
	3. How it helped to make a jump to become a tech lead.
```

would become

```
1. Introductory questions about overall background.
1.1. Music, computers and work at REDACTED.
1.2. Working with humans in non-tech
1.2.1. at work with REDACTED.
1.2.2. in an activist / advocacy group with REDACTED.
1.3. How it helped to make a jump to become a tech lead.
```

It doesn't look half-bad in Lagrange either:

![The same list rendered in Lagrange](https://user-images.githubusercontent.com/66186054/149591714-1aa1e314-514b-496f-b76a-43cb676e0580.png)

**Comment by n0x1m** [comment]
good call, what are the rules with new lines? Would be great to have a few input/output samples handy so we can play with the tests.
